oclint-rules/include/oclint/AbstractASTRuleBase.h

```diff
@@ -23,6 +24,7 @@ class AbstractASTRuleBase : public RuleBase
 
     void addViolation(const clang::Decl *decl, RuleBase *rule, const std::string& message = "");
     void addViolation(const clang::Stmt *stmt, RuleBase *rule, const std::string& message = "");
+    std::string declToStr(clang::Decl *decl);
```

Comment:
I am in favor of the idea, but suggest putting this one to a separate class other than RuleBase.

Reply:
Do you think it is better to put it into `AbstractASTVisitorRule` as a public method?

Reply:
Please move it to `https://github.com/oclint/oclint/blob/master/oclint-rules/include/oclint/util/ASTUtil.h`